### ISSUE TITLE
[Symfony] Tests try to insert duplicated data on 2.5.2

### ISSUE THREAD

**Issue by erussotf** [Codeception/Codeception#5362]
Hi. 

When upgrading from `2.5.1` to `2.5.2` many tests start failing because when try to insert data to DB, it fails because is duplicated.

The change that introduce this bug was this lines
https://github.com/Codeception/Codeception/blob/4d764d30e0be103cc6a78cee6fbbdd33dc90bc9e/src/Codeception/Lib/Connector/Symfony.php#L80-L82

### Details

* Codeception version: 2.5.x-dev#4d764d30e0be103cc6a78cee6fbbdd33dc90bc9e
* PHP Version: 7.1.23
* Installation type: Composer

Yml configuration:
```yml
modules:
    config:
        Db:
            dsn: ...
            user: ...
            password: ...
            dump: ...
            populate: true
            cleanup: false
```
If we set the `cleanup: true` it becomes like x10 slower, and do not solve all the tests.

**Comment by Naktibalda** [comment]
@erussotf I am still looking for volunteer to test my patch.
Please set composer constraint to `"codeception/codeception": "dev-symfony3-services"`, run `composer update` and then run your tests.

**Comment by yesdevnull** [comment]
We have the same error with tests failing due to DB rollbacks not being performed.  I'm not so sure that the Symfony module is to blame though.  Running @Naktibalda's `dev-symfony3-services` branch with our tests in debug mode never showed any messages relating to Symfony persisted services.

:heavy_check_mark: Codeception 2.5.1
:x: Codeception 2.5.2
:x: Codeception `dev-symfony3-services` (`e7317b4e628ed8aecbc90f5a72dcb48ece43d9e6`)

Our Symfony/DB modules config:
```yaml
modules:
  enabled:
    - Db:
        dsn: 'sqlsrv:Server=%TEST_DB_HOSTNAME%;Database=%TEST_DB_DATABASE%'
        user: '%TEST_DB_USERNAME%'
        password: '%TEST_DB_PASSWORD%'
        populate: true
        cleanup: true
        reconnect: true
    - Symfony:
        app_path: 'app'
        var_path: 'var'
        environment: 'test_acceptance'
        debug: false
        part: services
```

I haven't yet had the time to test properly, but my hunch is that [this commit](https://github.com/Codeception/Codeception/commit/486bf6a9ddd78832b60faaef9a6caade7a8eb4f4) is the root cause of the DB rollback issues a [few](https://github.com/Codeception/Codeception/issues/5345) [people](https://github.com/Codeception/Codeception/issues/5326) have been experiencing since upgrading to 2.5.2.

**Comment by Naktibalda** [comment]
@yesdevnull this ticket links 3 unrelated changes made in the last release:
1. Symfony persistent service change https://github.com/Codeception/Codeception/pull/5262
2. Doctrine cleanup change https://github.com/Codeception/Codeception/pull/5250/
3. Db don't clear database for empty dump https://github.com/Codeception/Codeception/pull/5211

Doctrine issues could be caused by Symfony or Db change, Db issues can only be caused by Db cleanup change.

**Comment by erussotf** [comment]
> @erussotf I am still looking for volunteer to test my patch.
> Please set composer constraint to `"codeception/codeception": "dev-symfony3-services"`, run `composer update` and then run your tests.

It seems to be working :smile: In which version will be released ? Maybe `2.5.3` ?

**Comment by Naktibalda** [comment]
Please upgrade to Codeception 2.5.3 and check if it works correctly.
